ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB/Model.hs

```diff
@@ -49,7 +49,7 @@ import           Ouroboros.Storage.Util.ErrorHandling (ThrowCantCatch)
 import qualified Ouroboros.Storage.Util.ErrorHandling as EH
 import           Ouroboros.Storage.VolatileDB.API
 import qualified Ouroboros.Storage.VolatileDB.Impl as Internal
-import           Ouroboros.Storage.VolatileDB.Util
+import           Ouroboros.Storage.VolatileDB.Util hiding (maxMaybe)
```

Comment:
So `maxMaybe` is defined here too? Replace it with `safeMaximum` too. If necessary use `max`, which works with two `Maybe a`s.